### DISCUSSION TITLE
The HTML code in the .vue file has comments and starts reporting an error. What's the reason?

### DISCUSSION THREAD

**Discussion by Zuojiangtao** [web-infra-dev/rsbuild#4303] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bf7da1f6-65ba-458a-9df9-a8aed5672052)

![image](https://github.com/user-attachments/assets/e493fa83-4650-4907-9350-02cdc5d57c65)

**Comment by chenjiahan** [comment]
Hi, can you provide a minimal reproduction repo?

**Reply by Zuojiangtao** [reply]
https://codesandbox.io/p/sandbox/rsbuild-bug-2hffxm

**Reply by chenjiahan** [reply, marked as answer]
This is because you have overridden all the built-in plugins and broken the vue-loader:

![image](https://github.com/user-attachments/assets/24cd3673-182e-4df5-8a07-7e0c456fb634)

You should push new plugins into the `plugins' array rather than overriding it:

![image](https://github.com/user-attachments/assets/7cc11ec4-6994-4671-9820-cdd7953ce5f2)

**Reply by Zuojiangtao** [reply]
我在plugins里声明的插件和tools里的plugins字段配置的插件不一样，怎么会冲突呢。

**Reply by chenjiahan** [reply]
The Rspack plugin and the Rsbuild plugin are independent. You are overwriting the internal Rspack plugin.

> Rspack 插件和 Rsbuild 插件是独立的，你覆盖的是内部的 Rspack 插件
